Sources/SmileID/Classes/SmileID.swift

```diff
@@ -26,21 +26,25 @@ public class SmileID {
     public static let version = "10.0.0-beta07"
     public private(set) static var config: Config!
     public private(set) static var useSandbox = true
+    internal static var apiKey: String?
     public private(set) static var theme: SmileIdTheme = DefaultTheme()
     internal private(set) static var localizableStrings: SmileIDLocalizableStrings?
     @ObservedObject internal static var router = Router<NavigationDestination>()
-
     /// This method initilizes SmileID. Invoke this method once in your applicaion lifecylce
     /// before calling any other SmileID methods.
     /// - Parameters:
+    ///   - apiKey: The api key displayed on your partner portal
     ///   - config: The smile config file. If no value is supplied, we check the app's main bundle
     ///    for a `smile_config.json` file.
     ///   - useSandbox: A boolean to enable the sandbox environment or not
-    public class func initialize(config: Config = try! Config(url: Bundle.main.url(forResource: "smile_config",
-                                                                                   withExtension: "json")!),
-                                 useSandbox: Bool = true) {
+    public class func initialize(
+        apiKey: String? = nil,
```

Comment:
This should be a new initialize function, because most people will not be using apiKey. IDE auto-complete etc would end up adding the parameter though, which may be confusing to partners

Thank you for adding in api key support